fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/LaoDetailViewModel.java

```diff
@@ -460,6 +512,14 @@ public Lao getCurrentLaoValue() {
     return mLaoEvents;
   }
 
+  public LiveData<List<com.github.dedis.student20_pop.model.RollCall>> getLaoRollCalls() {
+    return mLaoRollCalls;
```

Comment:
Instead of having a new variable you can filter the list of events to extract only the roll calls